lib/kafkalib/partition/settings.go

```diff
@@ -18,7 +18,7 @@ var ValidPartitionBy = []string{
 }
 
 type MergePredicates struct {
-	PartitionField string
+	PartitionField string `yaml:"partitionField"`
```

Comment:
Does this need a json annotation too?

Reply:
No it doesn't. I'm going to remove it from BigQuerySettings